<?xml version="1.0"?>
<launch>
    <node pkg="mdr_detect_emotion_action" type="detect_emotion_action" name="detect_emotion_server" output="screen" ns="mdr_actions">
        <param name="emotion_model_path" value="$(find mdr_detect_emotion_action)/common/config/emotion_classifier.hdf5" />
        <param name="detect_person_server" value="/mdr_actions/detect_person_server" />
        <param name="image_topic" value="/cam3d/rgb/image_raw"/>
    </node>
</launch>
